tests/bson/bson-utcdatetime-int-size.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 Test for UTCDateTime and integer parsing
+--SKIPIF--
+<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM implements only an older version of DateTime"); ?>
```

Comment:
In that case, it'd be better for Derick to clone this into the HHVM driver and mark it with XFAIL. Either way, I think we should periodically revisit skipped tests as HHVM improves to become more consistent with PHP.